cli/cmd/install.go

```diff
@@ -314,7 +310,12 @@ func renderCRDs(w io.Writer, values *l5dcharts.Values, valuesOverrides map[strin
 		return err
 	}
 
-	buf, _, err := renderChartToBuffer(files, values, valuesOverrides)
+	// The CRD chart does not take any value configuration.
+	values, err := l5dcharts.NewValues()
+	if err != nil {
+		return err
+	}
```

Comment:
Since the values struct is unused, can we just create a default one like `values := Values{}` rather than using `NewValues()`?

Reply:
It turns out that it's not totally unused, there's just nothing configurable. Specifically, things like:

```
{{- define "partials.annotations.created-by" -}}
linkerd.io/created-by: {{ .Values.cliVersion | default (printf "linkerd/helm %s" (.Values.cniPluginVersion | default .Values.linkerdVersion)) }}
{{- end -}}
```

In order to get the `cliVersion`, we need all of the values.

Really, I think we probably want to start decomposing the values struct, at least stripping out things like cliVersion.